vtm/src/org/oscim/layers/tile/bitmap/BitmapTileLayer.java

```diff
@@ -117,8 +138,13 @@ private void setFade(MapPosition pos) {
 
         float alpha = mBitmapAlpha;
         for (FadeStep f : fade) {
-            if (pos.scale < f.scaleStart || pos.scale > f.scaleEnd)
+            if (pos.scale < f.scaleStart) {
+                alpha = f.alphaStart;
                 continue;
+            } else if (pos.scale > f.scaleEnd) {
+                alpha = f.alphaEnd;
+                continue;
+            }
```

Comment:
If let interpolation to user, then those changes seem not needed.

Reply:
If you define one fadeStep, then you don't expect to set non fading steps too. That's why I posted this PR...